engine/config.go

```diff
@@ -115,6 +115,7 @@ func configBootstrap(args []string) Configuration {
 			conf.Hatchery.Openstack.HTTP.Port = 8086
 		case sdk.TypeHatchery + ":swarm":
 			conf.Hatchery.Swarm = &swarm.HatcheryConfiguration{}
+			conf.Hatchery.Swarm.Provision.InjectEnvVars = []string{"CDS=1"}
```

Comment:
To remove ?

Reply:
i think i can be an example, but i will remove it